setup.cfg

```diff
@@ -32,7 +32,7 @@ tests =
     flake8
     imageio
     matplotlib
-    hypothesis
+    hypothesis <=6.31.5
```

Comment:
This might fix the bug, but if I understand you correctly, the problem is primary the pointless distributions that are generated for testing. Could we somehow change those to get meaningful representations?